docs/DeveloperGuide.adoc

```diff
@@ -795,6 +971,26 @@ Private contact detail
 A contact detail that is not meant to be shared with others
 ....
 
+[[bright-coloured-mode]]
+Bright coloured mode
+....
+Using brighter colours for the user interface to make the AddressBook more visible under the sun
+....
+
+[[attribute]]
+Attribute
+
+....
+A characteristic of a person that is common across all people (e.g. Relationship/Marital Status, Birthday, Gender, Age)
+....
+
+[[suggested]]
+Suggested (a command)
+
+....
+To be suggested a command based on the detection of a misspelling of a command (e.g. find is suggested when fnid is spelled)
```

Comment:
May be not use the common word `suggested` here to mean a very specific thing? If you attribute a special meaning to a commonly used word, you will not be able to use that word for its usual meaning anymore.